src/auth/internal/AuthIntegrationInterface.ts

```diff
@@ -45,14 +45,16 @@ function isThereAnExistingSession(): boolean {
 async function inferAuthLogout() {
   if (!authState.client) return;
 
+  // We don't want to setAuthId('') here because user's can also revoke the
+  // session themselves with the auth client. So that should be performed in the
+  // auth client listeners
   if (isAuthStytch()) {
     await authState.client.session.revoke();
+    authState.onLogout();
```

Comment:
I think authState.onLogout should also be moved to the client listeners, but wanted to get your opinion